app/models/smart_proxy.rb

```diff
@@ -12,17 +12,17 @@ class SmartProxy < ActiveRecord::Base
   #TODO check if there is a way to look into the tftp_id too
   # maybe with a predefined sql
   has_and_belongs_to_many :features
-  has_many :subnets,                                          :foreign_key => 'dhcp_id'
```

Comment:
Please don't re-format files like this, revert changes that aren't to do with this PR.